firefox-ios/nimbus-features/microSurveyFeature.yaml

```diff
@@ -0,0 +1,19 @@
+# The configuration for the microSurveyFeature feature
+features:
+  micro-survey-feature:
+    description: >
+        A feature that shows the microsurvey for users to interact with and submit responses.
+    variables:
+      enabled:
+        description: >
+          If true, the feature is active.
+        type: Boolean
+        default: false
+
+    defaults:
+      - channel: beta
+        value:
+          enabled: false
+      - channel: developer
+        value:
+          enabled: false
```

Comment:
Q. should we enable it atleast in beta?

Reply:
I was thinking about enabling it when I implement the close button, so developers can easily dismiss it, without it being blocking.